### PR TITLE
Support `gunzip` in `fetch`

### PR DESCRIPTION
**Pull request by eitanturok**

### CODE REVIEW

**Comment by nextgens** [comment]
IMHO the goal is to test that gzip is enabled/working and that the API is stable rather than test gzip itself

That's the approach I went for in #6175

File: tinygrad/helpers.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 from __future__ import annotations
-import os, functools, platform, time, re, contextlib, operator, hashlib, pickle, sqlite3, tempfile, pathlib, string, ctypes, sys
+import contextlib, ctypes, functools, gzip, hashlib, operator, os, pathlib, pickle, platform, re, sqlite3, string, sys, tempfile, time
```
Comment (geohot) [inline tinygrad/helpers.py:2]: Why did this change?

File: test/unit/test_helpers.py [hunk 2]
```diff
@@ -161,6 +161,24 @@ def test_fetch_subdir(self):
       assert pimg.size == (77, 77), pimg.size
     assert img.parent.name == "images"
 
+  def test_fetch_gunzip_valid(self):
```
Comment (geohot) [inline test/unit/test_helpers.py:164]: Thank you, finally the correct test

File: tinygrad/helpers.py (hunk 1 above)
Comment (eitanturok) [reply tinygrad/helpers.py:2]: I sorted the imports alphabetically and added `gzip`.

Comment (chenyuxyz) [reply tinygrad/helpers.py:2]: but why?

Comment (eitanturok) [reply tinygrad/helpers.py:2]: I usually sort my imports.

LMK if you want me to do undo it.

**Comment by geohot** [comment]
email george@tinygrad.org with a PayPal or USDC (on ethereum) to claim $50

**Pull request merged**